src/PublicLab.Editor.js

```diff
@@ -42,15 +42,7 @@ PL.Editor = Class.extend({
 
       });
 
-      if (valid_modules == required_modules) {
-
-        $('.ple-publish').removeClass('disabled');
-
-      }
-
-      else {
-        $('.ple-publish').addClass('disabled');
-      }
+      $('.ple-publish').toggleClass('disabled', !(valid_modules == required_modules));
```

Comment:
Use the `prop` method not `addClass`

Reply:
I'm not sure if addClass works though.

Reply:
I am not using addClass, i have removed it. Instead i used the toggleClass method to achieve the same functionality as the previous if...else block

Reply:
Toggle class will just toggle the class and ignore the second parameter. Correct me if I am wrong.